src/components/filter/FilterWrapper.module.scss

```diff
@@ -27,24 +27,28 @@ $border-right-color: var(--amino-filter-wrapper-border-right-color);
   cursor: pointer;
   display: flex;
   align-items: center;
-  gap: theme.$amino-space-8;
-  padding: theme.$amino-space-4 theme.$amino-space-8;
+  gap: theme.$amino-space-0;
+  padding: theme.$amino-space-4 theme.$amino-space-4;
   border-radius: 100px;
   border-top-right-radius: $border-top-right-radius;
   border-bottom-right-radius: $border-bottom-right-radius;
-  border-right: 1px solid $border-right-color;
+  border-right: 0px solid $border-right-color;
```

Comment:
This seems to have no effect here. `border` with 0 width means no border. 🤔 If that's what you mean, you can just remove this line.